### PR TITLE
Refactor '/aim/web/api' directory

### PR DESCRIPTION
**Pull request by mihran113**
- Cleanup old not used views, models and utils
 - Change paths for run structured data endpoints
 - Minor fixes for endpoints after testing
 - Minor fixes run.track() method
 - Moved old utils that are still needed for storage files conversion to 'aim/web/legacy_api'

### CODE REVIEW

File: aim/web/api/projects/views.py [hunk 1]
```diff
@@ -25,9 +24,8 @@ async def project_api():
     return {
         'name': project.name,
         'path': project.path,
-        'tf_enabled': project.tf_enabled,
         'description': project.description,
-        'branches': project.repo.list_branches(),
+        'branches': factory.experiments(),
```
Comment (alberttorosyan) [inline aim/web/api/projects/views.py:28]: @mihran113 here you should iterate over `factory.experiments()` and get dicts from it. Take a look to `web/api/experiments/views.py:get_experiments_list_api` for reference.

**Review by alberttorosyan — COMMENTED** [review]
Great job! Please fix one comment and merge.

File: aim/web/api/projects/views.py (hunk 1 above)
Comment (mihran113) [reply aim/web/api/projects/views.py]: done ✔️

**Review by gorarakelyan — APPROVED** [review]
lgtm

**Pull request merged**